public/fonts/satoshi/css/satoshi.css

```diff
@@ -0,0 +1,178 @@
+/**
```

Comment:
Is this font available to use for commercial purposes? Please verify we can use this, and any restrictions/stipulations.

Reply:
this font was chosen by Sasha in the figma design

Reply:
> Is this font available to use for commercial purposes? Please verify we can use this, and any restrictions/stipulations.

@pi0neerpat We can use it. https://www.fontshare.com/licenses/itf-ffl